GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+
+Contributors are community members who contribute in concrete ways to the project,
+most often in the form of code and/or documentation. Anyone can become a Contributor,
+and contributions can take many forms, e.g.:
+* Help other users through any of the [communication channels](#communication-channels)
+made for that purpose.
+* Triage GitHub issues.
+* Organize Selenium meetups.
+* Organize and collaborate in Selenium Conferences.
+
+There is no expectation of commitment to the project, no specific skill requirements,
+and no selection process.
+
+Some Contributors might have some basic privileges to the GitHub repos, based on
+their type of contribution (e.g., close an issue after triaging it).
+
+Contributors have read-only access to source code and submit changes via pull
+requests. Contributor pull requests have their contribution reviewed and merged
+by a TLC member or a Committer. TLC members and Committers work with Contributors
+to review their code and prepare it for merging.
+
+As Contributors gain experience and familiarity with the project, their profile within,
+and commitment to, the community will increase. At some stage, they may find themselves
+being nominated for the Committer role by an existing Committer or TLC member.
+
+### Project Committers
+
+Committers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the community.
+Committers are given push access to the project's GitHub repos where they
+contribute to.
+
+Committers:
+
+* Should bring their proposals for large changes to the project's code first to
+a GitHub issue, and all relevant committers should be pinged so they can weigh
+in on the discussion if desired. 
+* Debates between committers about whether code should be merged should happen
+in GitHub pull requests to keep the project decision history.
+* In general any committer can review and merge a pull request. Committers
+should only merge code they are qualified to review, which might entail pinging
+another committer who has greater ownership over a specific code area.
+* May label and close issues.
+
+To become a Committer:
+
+* One must have shown a willingness and ability to participate in the project as
+a team player.
+* Committers are expected to be respectful of every community member and to work
+collaboratively in the spirit of inclusion.
+* Have submitted a minimum of 10 qualifying pull requests. Where significant
+technical weight is present and requires little effort to accept because it is
+well documented and tested.
+
+New Committers can be nominated by any existing Committer or TLC member. Once
+they have been nominated, there will be a vote by the TLC members.
+
+It is important to recognize that having the Committer role is a privilege, not
+a right. That privilege must be earned and once earned, it can be removed by
+the TLC members by a standard TLC motion. However, under normal circumstances
```

Comment:
from a personal situation perspective (a committer who has not been active in a few years). Should this be a little more explicit on losing the privilege? @shs96c has before said it's a seemingly permanent privilege grant and this seems like it's language for those that might be violating some kind of Code of Conduct?

Like, am I going to just lose it someday unbeknownst to myself until github tells me I've been removed? Or is there plans to have communication with the person? ( "it can be removed by the TLC members by a standard TLC motion" seems not very well defined of a process, especially if one objects?)

Reply:
As you pointed out, the intent of this section is to remove the committing privilege in case the committer violates some kind of Code of Conduct. Since we do not have it yet, I left it open on purpose (there is a separate stream where the Code of Conduct is being put together).

In general, I don't think that the commit bit should be removed if the committer has been inactive. We cannot just kick out people who have contributed so much from their time to the project, which is in line with what @shs96c has said before. 

Do you think we should make this section more specific? What do you suggest @lukeis?

Reply:
I don't have any specific recommendations for now. When a CoC is created / published, this section should be updated with references of that intent.

Thanks for you for confirming the unofficial policy 😄

Reply:
👍